Add a new swagger json file for Express Route Transport

dedhar opened the pull request:
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

dedhar committed:
Add a new swagger json file for Express Route Transport

msftclas commented:
@__dedhar__,
Thanks for your contribution.
To ensure that the project team has proper rights to use your work, please complete the Contribution License Agreement at https://cla.microsoft.com.

It will cover your contributions to all Microsoft-managed open source projects.
_Thanks,
Microsoft Pull Request Bot_

dsgouda reviewed (changes requested):
There are multiple keys on lines 179 and 241. Please fix these errors before I can review.

dedhar committed:
Resolve Travis CI build failures

dedhar committed:
Added examples for all Express Route Transport Operations

dedhar commented:
Addressed Deepak’s comments , fixed build break and added examples.

Thanks
Deepal

From: Deepak Shankargouda [mailto:notifications@github.com]
Sent: Monday, July 17, 2017 5:42 PM
To: Azure/azure-rest-api-specs <azure-rest-api-specs@noreply.github.com>
Cc: Deepal Dhariwal <dedhar@microsoft.com>; Author <author@noreply.github.com>
Subject: Re: [Azure/azure-rest-api-specs] Add a new swagger json file for Express Route Transport (#1430)


@dsgouda requested changes on this pull request.

There are multiple keys on lines 179 and 241. Please fix these errors before I can review.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F1430%23pullrequestreview-50477903&data=02%7C01%7Cdedhar%40microsoft.com%7Cb88764b6730444c4b70508d4cd75d37c%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636359353338193286&sdata=VhqYEeP8yzRBhW0NbSbFoKg%2BkRWecNYAm%2Fxws6oOW%2BM%3D&reserved=0>, or mute the thread<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAP4nMepPGNyxxn5oTJb53QKNWg6HzicVks5sO_FSgaJpZM4OanmN&data=02%7C01%7Cdedhar%40microsoft.com%7Cb88764b6730444c4b70508d4cd75d37c%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636359353338193286&sdata=LsAdBMbETa%2BgsF40pTv2igFtraRDUXb%2BChLRZyVL1Ws%3D&reserved=0>.

dedhar commented:
Addressed comments, Fix Travis CI Build Failures and added examples

dedhar committed:
Fixed Travis CI build errors in example json files

dedhar committed:
Fix of the stream or a document separator in ExpressRoutetransportGet.json

fearthecowboy commented:
@dedhar -- we've reorganized this repository a bit -- can I get you to make a clean fork of the repository, and base your PR off the `current` branch instead of `master, and then issue a new PR?

// cc @salameer
